### ISSUE TITLE
[Forms] Forms are without buttons?

### ISSUE THREAD

**Issue by javac9** [composablehorizons/composablesui#19]
It would be nice to have an example (or all of them) with some buttons (Save/Submit, Reset/Discard, etc.)

**Comment by alexstyl** [comment]
Thanks for opening this issue. The forms section definitely needs a retouch.

Will prioritize this work to get it out asap.

If the meantime if you have some specific use case you want to see, let me know.

**Comment by alexstyl** [comment]
I agree. The forms are probably one of the oldest things implemented in compose examples.

The way of writing you described was introduced a bit later. 

For example have a look at : https://composeexamples.com/components/application-ui/screens/signup where the data is separated to the UI.

The new forms will be written in a similar fashion.
